load-test/README.md

```diff
@@ -19,6 +19,12 @@
 * Download [JMeter 4.0](http://www-us.apache.org/dist//jmeter/binaries/apache-jmeter-4.0.tgz)
 * Run `/bin/jmeter.bat` to launch JMeter
 
+### Install Jmeter Plugins - Custom JMeter Functions
+* download Jmeter plugin manager
+https://www.blazemeter.com/blog/how-install-jmeter-plugins-manager?utm_source=blog&utm_medium=BM_blog&utm_campaign=how-to-use-the-custom-jmeter-functions-plugin-part2
```

Comment:
very minor, but you can remove everything after the `?`, as its just tracking data.
link should be `https://www.blazemeter.com/blog/how-install-jmeter-plugins-manager`

Reply:
k..will update the readme